src/manifest/msquic.clog_config

```diff
@@ -75,6 +79,138 @@
     ]
   },
   "SourceCodeMacros": [
+    {
+      "MacroName": "QuicTraceLogWarning",
+      "EncodedPrefix": null,
+      "EncodedArgNumber": 1,
+      "MacroConfiguration": {
+        "linux": "lttng_plus",
```

Comment:
I feel like we should rename `lttng_plus` back to just `lttng` since we're making this official.